Library/Homebrew/utils.rb

```diff
@@ -64,7 +64,8 @@ def width
     end
 
     def truncate(str)
-      str.to_s[0, width - 4]
+      width_ = width
+      width_ > 10 ? str.to_s[0, width_ - 4] : str
```

Comment:
Couldn't we just do `return str if width <= 10` as the first line in this function?